air/src/proof/security.rs

```diff
@@ -24,19 +24,16 @@ const MAX_PROXIMITY_PARAMETER: u64 = 1000;
 pub struct ConjecturedSecurity(u32);
 
 impl ConjecturedSecurity {
-    /// Computes the security level (in bits) of the protocol using a modification of Eq. (19) in
-    /// [1].
+    /// Computes the security level (in bits) of the protocol using Eq. (19) in [1].
     ///
     /// [1]: https://eprint.iacr.org/2021/582
     pub fn compute(
         options: &ProofOptions,
         base_field_bits: u32,
-        trace_domain_size: usize,
         collision_resistance: u32,
     ) -> Self {
         // compute max security we can get for a given field size
-        let field_size = base_field_bits * options.field_extension().degree();
-        let field_security = field_size - (trace_domain_size * options.blowup_factor()).ilog2();
+        let field_security = base_field_bits * options.field_extension().degree();
```

Comment:
This makes conjectured security estimate a bit less conservative, right?

Reply:
Indeed, the conjecture implies that the soundness of the protocol is not affected by the domain size or, as far as I can see, the batching used in either the computation of the quotient polynomials or the DEEP polynomial.